src/rata-run-workspace/rata-run.service.ts

```diff
@@ -31,4 +40,33 @@ export class RataRunWorkspaceService {
 
     return this.map.one(result);
   }
+
+  async createRataRun(
+    testSumId: string,
+    rataSumId: string,
+    payload: RataRunBaseDTO,
+    userId: string,
+    isImport: boolean = false,
+  ): Promise<RataRunRecordDTO> {
+    const timestamp = currentDateTime().toLocaleDateString();
```

Comment:
```suggestion
    const timestamp = currentDateTime();
```
Does not need to convert to string?